js/h5p.js

```diff
@@ -1400,6 +1406,145 @@ H5P.openEmbedDialog = function ($element, embedCode, resizeCode, size, instance)
   dialog.open();
 };
 
+
+/**
+ * Internal recursive function that determines if two objects are equivalent
+ * 
+ * Returns true if equal and false otherwise
+ * 
+ * @param {object} aObject First object being compared
+ * @param {object} bObject Second object being compared
+ * @param {Array} aStack a stack reprentation of the first objects' internal object stack
```

Comment:
You can mark a parameter as optional in JSdoc by putting its identifier in []'s

e.g. 
`@param {Array} [aStack] a stack reprentation of the first objects' internal object stack`